testgrid/web/src/components/views/InstanceTable.jsx

```diff
@@ -36,20 +36,20 @@ const InstanceTable = (props) => {
   const [showPostInstallScript, setShowPostInstallScript] = useState(false);
   const [showPostUpgradeScript, setShowPostUpgradeScript] = useState(false);
 
-  // TODO: this does not work
+  let logsAceEditor = null;
+
```

Comment:
future note to refactor this to not set the input directly in the AceEditor but for now 🤷‍♀️